docs/ref/settings.txt

```diff
@@ -791,7 +791,8 @@ on :ref:`controlling the creation order of test databases
 
 Default: ``True``
 
-When set to ``False``, migrations won't run when creating the test database.
+When set to ``False``, migrations won't run when creating the test database
+(tables will still be created for the apps' models).
```

Comment:
Just wonder is it's worth linking here to :option:`migrate --run-syncdb` or :setting:`MIGRATION_MODULES`? 

`Similar to ...`

Reply:
I added: ```Same as when setting ``None`` as a value in :setting:`MIGRATION_MODULES`, but for all apps.```.